### ISSUE TITLE
Jq Transformer, transformation setting

### ISSUE THREAD

**Issue by Robertbaelde** [vudash/vudash#46]
When using the Jq transformer, its documented that one should use the following config for a transformation
```
"transformations": [
    {
      "transformer": "@vudash/transformer-jq",
      "options": ".a.big.json | [\"empty\", .[1], \"useless\", .[3]] | join(\" \")"
    }
  ],
```

However, this results in a invalid configuration, as there should be a value key in the options object.
I propose to change the configuration of the JQ transformer to 
```
"transformations": [
    {
      "transformer": "@vudash/transformer-jq",
      "options": {
            value: ".a.big.json | [\"empty\", .[1], \"useless\", .[3]] | join(\" \")"
        }
    }
  ],
```
And then in the transformer.js use `this.transformation = transformation.value`

Another solution would be to remove the required value from the configuration validation.

**Comment by antony** [comment]
I think I agree with this. It's probably more logical if options follows a standard format.

**Comment by antony** [comment]
This has been fixed as part of 9.2.0
